source/_components/binary_sensor.ihc.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: "IHC Binary Sensor"
+description: "Instructions how to integrate IHC Binary Sensors within Home Assistant."
+date: 2017-11-27 13:35
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ihc.png
+ha_category: Binary Sensor
+ha_release: 0.59
+ha_iot_class: "Local Push"
+---
+
+Before you can use the IHC Binary Sensor platform, you must setup the [IHC Component](../ihc/)
```

Comment:
Does the relative link work?

Reply:
It does work because the url has an ending / but I have changed all links to absolute /components/... (I can see that is how it is done on the other pages)